dnf-behave-tests/features/plugins-core/config-manager.feature

```diff
@@ -14,6 +14,15 @@ Background:
 Scenario: when run without arguments
    When I execute dnf with args "config-manager"
    Then the exit code is 1
+    And stdout contains "usage: dnf config-manager"
```

Comment:
Hm, this message actually depends on the command under which the tests run (dnf/yum).